### PR TITLE
Add missing paddings in Cosmos structures

### PR DESCRIPTION
**Pull request by pienkowb**

### CODE REVIEW

**Comment by roignpar** [comment]
do we know if these types that now have padding actually work? (don't panic when calling `to_asc_bytes` on them)

**Comment by pienkowb** [comment]
> do we know if these types that now have padding actually work? (don't panic when calling `to_asc_bytes` on them)

@roignpar I think we need to test that with an instrumented Osmosis node.

**Comment by Seb-figment** [comment]
> > do we know if these types that now have padding actually work? (don't panic when calling `to_asc_bytes` on them)
> 
> @roignpar I think we need to test that with an instrumented Osmosis node.

or with unit test.

**Comment by roignpar** [comment]
> > > do we know if these types that now have padding actually work? (don't panic when calling `to_asc_bytes` on them)
> > 
> > 
> > @roignpar I think we need to test that with an instrumented Osmosis node.
> 
> or with unit test.

https://github.com/figment-networks/graph-node/pull/33

File: chain/cosmos/src/runtime/abi.rs [hunk 1]
```diff
@@ -273,8 +273,8 @@ impl ToAscObj<AscLightClientAttackEvidence> for codec::LightClientAttackEvidence
             conflicting_block: asc_new_or_null(heap, &self.conflicting_block, gas)?,
             _padding: 0,
             common_height: self.common_height,
-            byzantine_validators: asc_new(heap, &self.byzantine_validators, gas)?,
             total_voting_power: self.total_voting_power,
+            byzantine_validators: asc_new(heap, &self.byzantine_validators, gas)?,
```
Comment (roignpar) [inline chain/cosmos/src/runtime/abi.rs:277]: the order of the fields is only relevant when defining the struct. it makes no difference when instantiating it.

Comment (pienkowb) [reply chain/cosmos/src/runtime/abi.rs:277]: I wanted to keep the order consistent everywhere :)

**Review by Seb-figment — APPROVED** [review]
LGTM

File: chain/cosmos/src/runtime/generated.rs [hunk 2]
```diff
@@ -480,6 +482,7 @@ impl Default for AscSignedMsgType {
 #[derive(AscType)]
 pub(crate) struct AscLightClientAttackEvidence {
     pub conflicting_block: AscPtr<AscLightBlock>,
+    pub _padding: u32,
```
[the remark this answers is not in the document]
Comment (Seb-figment) [reply chain/cosmos/src/runtime/generated.rs:485]: Just saw that my comment where pending until this morning. Sorry about that. :-)

**Pull request merged**